### PR TITLE
Rework `EditImageDialog`

### PR DESCRIPTION
**Pull request by fichtnerma**
Changes

- Increase image size
- Add hover effects for focal points
- Add "Open in DAM" button

---

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: COM-850
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
    
image

</details>

### CODE REVIEW

File: packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx [hunk 1]
```diff
@@ -46,6 +51,7 @@ interface Props {
         height: number;
         size?: number;
     };
+    damFileId?: string;
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx:54]:
```suggestion
    damFileId: string;
```

This can't be undefined

File: packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx [hunk 2]
```diff
@@ -149,7 +159,7 @@ export function EditImageDialog({ image, initialValues, onSubmit, onClose, inher
                             <div>
                                 {inheritedDamSettings !== undefined && (
                                     <>
-                                        <Box padding={8}>
+                                        <Box padding={8} paddingBottom={6}>
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx:162]: Does it look alright if the dependencies aren't configured? Or should the padding be added to the box below?

File: packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx [hunk 3]
```diff
@@ -168,6 +178,28 @@ export function EditImageDialog({ image, initialValues, onSubmit, onClose, inher
...
                                         </Box>
+
+                                        {dependencyMap["DamFile"] && damFileId && (
+                                            <Box padding={7} paddingTop={0}>
+                                                <Button
+                                                    variant="text"
+                                                    color="inherit"
+                                                    onClick={async () => {
+                                                        // id is checked three lines above
+                                                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx:189]:
```suggestion
```

There is no non-null-assertion here and the check isn't three lines above, but six. Please don't copy/paste code without thinking 😁

File: packages/admin/cms-admin/src/common/image/ImageCrop.sc.ts [hunk 4]
```diff
@@ -78,12 +87,16 @@ export const FocalPointHandle = styled("button")<FocalPointHandleProps>`
         border-radius: 50%;
         background-color: ${({ theme }) => theme.palette.primary.main};
         margin: 9px;
+        /* transform: translate(-50%, -50%); */
+        /* left: 50%; */
+        /* top: 50%; */
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/common/image/ImageCrop.sc.ts:92]:
```suggestion
```

File: packages/admin/cms-admin/src/common/image/ImageCrop.sc.ts [hunk 5]
```diff
@@ -78,12 +87,16 @@ export const FocalPointHandle = styled("button")<FocalPointHandleProps>`
...
+        transition: all 0.5s ease;
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/common/image/ImageCrop.sc.ts:93]: What do you want to animate? Please specify the respective property instead of "all"

File: packages/admin/cms-admin/src/common/image/ImageCrop.tsx [hunk 6]
```diff
@@ -36,6 +36,8 @@ export const ImageCrop = (props: ImageCropProps): React.ReactElement => {
                         crop={disabled ? {} : { ...value, unit: "%" }}
                         ruleOfThirds={true}
                         disabled={disabled}
+                        style={{ marginLeft: "40px" }}
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/common/image/ImageCrop.tsx:39]: Why is this necessary? And shouldn't it be a padding on ImageContainer instead?

File: packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx (hunk 2 above)
Comment (fichtnerma) [reply packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx:162]: image

Looks good to me

File: packages/admin/cms-admin/src/common/image/ImageCrop.tsx (hunk 6 above)
Comment (fichtnerma) [reply packages/admin/cms-admin/src/common/image/ImageCrop.tsx:39]: It is used to align the image on the top left corner

Comment (johnnyomair) [reply packages/admin/cms-admin/src/common/image/ImageCrop.tsx]: We removed the padding for the DialogContent [here](https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/blocks/image/EditImageDialog.tsx#L58-L62) to achieve the previous, now outdated design. Maybe it's a good idea to re-add it?

**Review by johnnyomair — APPROVED** [review]
